st2actions/st2actions/handlers/mistral.py

```diff
@@ -40,6 +43,15 @@ def get_handler():
     return MistralCallbackHandler
 
 
+def get_action_execution_id_from_url(url):
+    match = re.search('(.+)/action_executions/(.+)', url)
+    if not match or len(match.groups()) != 2:
+        raise ValueError('Unable to extract the action execution ID '
+                         'from the callback URL (%s).' % url)
```

Comment:
Minor style thing - it's safer to put parenthesis around the format string arguments (`% (url)`) in case we add more arguments later.